### DISCUSSION TITLE
Termux update issue

### DISCUSSION THREAD

**Discussion by Jerryachayath** [termux/termux-app#2604] [General]
Kindly solve the problem anyone
Get:1 https://grimler.se/termux-packages-24 stable InRelease [16.8 kB]
Get:2 https://grimler.se/game-packages-24 games InRelease [9347 B]                                              Get:3 https://grimler.se/science-packages-24 science InRelease [9360 B]                                         Err:1 https://grimler.se/termux-packages-24 stable InRelease                                                      The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B0076E490B71616B
Err:2 https://grimler.se/game-packages-24 games InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B0076E490B71616B                                                     Err:3 https://grimler.se/science-packages-24 science InRelease                                                    The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B0076E490B71616B
Reading package lists... Done                           W: GPG error: https://grimler.se/termux-packages-24 stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B0076E490B71616B                                         E: The repository 'https://grimler.se/termux-packages-24 stable InRelease' is not signed.                       N: Metadata integrity can't be verified, repository is disabled now.                                            N: Possible cause: repository is under maintenance or down (wrong sources.list URL?).                           W: GPG error: https://grimler.se/game-packages-24 games InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B0076E490B71616B
E: The repository 'https://grimler.se/game-packages-24 games InRelease' is not signed.
N: Metadata integrity can't be verified, repository is disabled now.
N: Possible cause: repository is under maintenance or down (wrong sources.list URL?).
W: GPG error: https://grimler.se/science-packages-24 science InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B0076E490B71616B
E: The repository 'https://grimler.se/science-packages-24 science InRelease' is not signed.
N: Metadata integrity can't be verified, repository is disabled now.
N: Possible cause: repository is under maintenance or down (wrong sources.list URL?).

**Comment by agnostic-apollo** [comment]
Run `termux-change-repo` and switch to a different `main` mirror, then run `apt install termux-tools termux-keyring`

**Comment by 2096779623** [comment]
`apt-key adv --recv B0076E490B71616B`
